### PR TITLE
Lightfire is a Fluffy IPC

### PR DESCRIPTION
**Pull request by Aurorablade**
Fluff items via Paetron Backer tier
1: classy victorian suit
2: classy bowler hat

Ckey: Lightfire
Character:Hyperion
Picture:

:cl: Fethas
rscadd:Lightfire Fluff items added.
/:cl:

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -1246,3 +1246,19 @@
...
+	icon_state = "bowler_lightfire"
+
+/obj/item/clothing/under/fluff/lfvicsuit //Lightfire: Hyperion
+	name = "Classy victorian suit"
+	desc = "A purple and black victorian suit with silver buttons, very fancy!"
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "victorianlightfire"
+	item_state = "victorianvest"
+	item_color = "victorianlightfire"
+	displays_id = 0
```
Comment (Regen1) [inline code/modules/customitems/item_defines.dm]: Why?

I'm not 100% sure what it does, but why is the var specified?

Comment (Aurorablade) [reply code/modules/customitems/item_defines.dm]: i saw it on a few others of this type, but i may hide the id badge icon on jumpsuits on mob. I think

Comment (Regen1) [reply code/modules/customitems/item_defines.dm]: https://github.com/Aurorablade/Paradise/blob/70963a8a42cadd2b08ae01064cd0e8a081ccc027/code/modules/customitems/item_defines.dm#L1062

Seems to be alright

Comment (taukausanake) [reply code/modules/customitems/item_defines.dm]: I don't think it does anything? I did a grep for it and only got assignments for its value. It's not used in any procs as far as I can tell. It's also used in the Centcom uniforms in miscellaneous.dm so maybe that means something

Comment (LightFire53) [reply code/modules/customitems/item_defines.dm]: Please, fethas, it's blue and black, not purple

Comment (Aurorablade) [reply code/modules/customitems/item_defines.dm]: it looks purple on my screen....*stares*

Comment (LightFire53) [reply code/modules/customitems/item_defines.dm]: https://i.gyazo.com/6916211cb0203d9b9096891d90e3bd76.png

Comment (taukausanake) [reply code/modules/customitems/item_defines.dm]: RGB code (the color in the middle): R: 46 G: 39 B: 120

Looks purple to me

Comment (Fox-McCloud) [reply code/modules/customitems/item_defines.dm]: `FALSE`

**Pull request merged**